### DISCUSSION TITLE
doubt about styles

### DISCUSSION THREAD

**Discussion by Eliasjk17** [lllyasviel/Fooocus#3455] [Q&A] [answered]
My question is basically about the styles within the application. Are these styles also subject to permission from their creator for commercial use, like Models or LoRA? Or can the styles be used without any type of restriction for commercial use?
I recently learned to check if Models and LoRA could be used with commercial intentions, and when implementing these models in Fooocus I saw that a new style was added to the list of styles (there is my doubt) and I was trying to search for the styles but I didn't find anything on this topic. To be more specific, the styles that interest me are: Fooocus V2, Fooocus Negative, Fooocus Photograph, Cinematic Diva and Photo Iphone Photographic.

**Comment by mashb1t** [comment, marked as answer]
Styles are exactly the same as adding text prompts, except Fooocus V2, which is the prompt expansion using GPT2 locally.
Fooocus V2 can be used within Fooocus for commercial interests but is not permitted to be used outside of Fooocus for commercial use, see https://github.com/lllyasviel/Fooocus/blob/main/extras/expansion.py#L4.

**Reply by Eliasjk17** [reply]
So, I can't take the program file that contains fooocus v2 and use it outside of fooocus, but I can generate an image with fooocus v2 and put that image to commercial use, do I understand that right?

**Reply by mashb1t** [reply]
Given you use models/LoRAs which grant you commercial usage: Yes.
See https://github.com/lllyasviel/Fooocus/discussions/3362#discussioncomment-10156210

**Reply by Eliasjk17** [reply]
I agree, thank you, Fooocus seems to me to be a magnificent tool and an opportunity for anyone to improve their life with some extra income, but above all I did not want to have problems with anyone for using it inappropriately without knowing that they were doing something wrong, thank you .
